src/workloads/execution/UnshardedLookup.yml

```diff
@@ -0,0 +1,70 @@
+SchemaVersion: 2018-07-01
+Owner: "@mongodb/query"
+Description: |
+  Benchmark a $lookup from one unsharded collection to another in a sharded environment.
+
+Actors:
+- Name: InsertData
+  Type: Loader
+  Threads: 1
+  Phases:
+  - Repeat: 1
+    Database: &db test
+    Threads: 1
+    DocumentCount: 1000
+    BatchSize: &batchSize 1000
+    CollectionCount: 2
+    Document:
+      shardKey: {^RandomInt: {min: 1, max: 1000}}
+      a: {^RandomInt: {min: 1, max: 1000}}
+      b: &string {^RandomString: {length: 10000}}
+  - &Nop {Nop: true}
+  - *Nop
+
+- Name: Quiesce
+  Type: RunCommand
+  Threads: 1
+  Phases:
+  - *Nop
+  - Repeat: 1
+    Database: admin
+    Operations:
+    - OperationName: RunCommand
+      OperationCommand:
+        fsync: 1
+  - *Nop
+
+- Name: Lookup
+  Type: RunCommand
+  Database: *db
+  Phases:
+  - *Nop
+  - *Nop
+  - Repeat: 10
+    Database: *db
+    Operations:
+    - OperationMetricsName: Lookup
+      OperationName: RunCommand
+      OperationCommand:
+        aggregate: Collection0
+        pipeline:
+          [{
+            $lookup: {
+              from: "Collection1",
+              let: {a0: "$a"},
+              pipeline: [{
+                $match: {
+                  $expr: {$eq: ["$a", "$$a0"]},
```

Comment:
If 'a' is a random int from 1-1000, and we only have 1000 documents (is this for each collection or in total?) with an even distribution there won't be many "hits" right? Aka each doc flowing into the $lookup stage will on average only match one result. Can we add another phase to measure the performance for when we'd have to send more results over the network?

Reply:
Yes that's correct, on average we'd expect 1 match per document. Note each document has a 10KB string field in order to make sure we're sending a non-trivial payload through the network interface. But I can certainly add another benchmark with a greater payload if you think it would be useful.

Reply:
Ah got it, I didn't see that the 'c' field was so big - thanks for that clarification!